### PR TITLE
Feature fsrs loader

### PR DESCRIPTION
**Pull request by alburde1**
Script to load FSRS subgrant and subcontract data from broker.

Technical Notes:
- Migration for award_subaward table
- Running script: `python manage.py load_fsrs`

### CODE REVIEW

File: usaspending_api/broker/management/commands/load_fsrs.py [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+        return dictfetchall(db_cursor)
+
+    @staticmethod
+    def get_award(row, award_type):
+        if award_type == 'procurement':
+            # we don't need the agency for grants
+            agency = get_valid_awarding_agency(row)
+
+            if not agency:
+                # logger.warning(
```
Comment (nmonga91) [inline usaspending_api/broker/management/commands/load_fsrs.py]: Is this warning no longer necessary? if it's not, can we just remove it as well the one below that's similar?

Comment (alburde1) [reply usaspending_api/broker/management/commands/load_fsrs.py]: Crud, I knew I'd forgotten to uncomment something. I commented them out because they were causing too many warnings locally and I wanted to test. We want them printing so we know what internal IDs had problems

File: usaspending_api/broker/management/commands/load_fsrs.py [hunk 2]
```diff
@@ -108,10 +109,15 @@ def gather_shared_award_data(self, data, award_type):
...
-            )
+            try:
+                recipient, created = LegalEntity.objects.get_or_create(
+                    recipient_unique_id=row['duns'],
+                    recipient_name=recipient_name
+                )
+            except MultipleObjectsReturned:
+                created = False
+                print('Legal Entity with DUNS: {} and name: {} returned two rows. '
+                      'Skipping...'.format(row['duns'], recipient_name))
```
Comment (alburde1) [inline usaspending_api/broker/management/commands/load_fsrs.py]: This isn't actually skipping anything, you need to put a `continue` after this to skip this record. If you want to get the first one you have to do something else. If you want a NULL in this case you have to set recipient to NULL

File: usaspending_api/broker/management/commands/load_fsrs.py [hunk 3]
```diff
@@ -108,10 +109,15 @@ def gather_shared_award_data(self, data, award_type):
...
-                recipient_name=recipient_name
-            )
+            try:
+                recipient, created = LegalEntity.objects.get_or_create(
+                    recipient_unique_id=row['duns'],
+                    recipient_name=recipient_name
+                )
+            except MultipleObjectsReturned:
+                created = False
+                print('Legal Entity with DUNS: {} and name: {} returned two rows. '
```
Comment (alburde1) [inline usaspending_api/broker/management/commands/load_fsrs.py]: This should be a logger not a print

File: usaspending_api/broker/management/commands/load_fsrs.py [hunk 4]
```diff
@@ -68,9 +68,9 @@ def get_award(row, award_type):
 
             # We don't have a matching award for this subcontract, log a warning and continue to the next row
             if not award:
-                logger.warning(
-                    "Internal ID {} cannot find award with piid {}, parent_award_id {}; skipping...".
-                    format(row['internal_id'], row['contract_number'], row['idv_reference_number']))
+                #logger.warning(
+                #    "Internal ID {} cannot find award with piid {}, parent_award_id {}; skipping...".
+                #    format(row['internal_id'], row['contract_number'], row['idv_reference_number']))
```
Comment (alburde1) [inline usaspending_api/broker/management/commands/load_fsrs.py]: This shouldn't be commented out forever, not sure what the plan is in the end but we need to remember to uncomment at some point.

File: usaspending_api/etl/helpers.py [hunk 5]
```diff
@@ -78,12 +78,15 @@ def get_or_create_location(row, mapper):
...
     if country_code and country_code.country_code == "USA":
+        
         # Apparently zip codes are optional...
         if location_dict["location_zip"]:
             location_dict.update(
                 zip5=location_dict["location_zip"][:5],
                 zip_last4=location_dict["location_zip"][5:])
-            location_dict.pop("location_zip")
+
+        location_dict.pop("location_zip")
```
Comment (alburde1) [inline usaspending_api/etl/helpers.py]: Why was this moved out of the `if`? Do we always want to pop even if it's NULL?
